README.md

```diff
@@ -100,15 +100,15 @@ Moja global welcomes contributions to the community website. If you have an idea
 
 ### Linting
 
-The project has ESLint configured to properly format the code. This enables developer to follow the same coding style and to avoid improperly-linted code. All the commands below should be run from the `website` directory.
+The project has ESLint configured to properly format the code. This enables developers to follow the same coding style and to avoid improperly-linted code. All the commands below should be run from the `website` directory.
 
 ```sh
 yarn lint
 ```
-This command logs all warnings and lint errors and scope of fixes/improvements.
+This command logs all warnings and lint errors and the scope of fixes/improvements.
 
 ```sh
-yarn lint:fix
+yarn lint: fix
```

Comment:
This is a command `yarn lint:fix`. We can remove the space in between the command.